setup.cfg

```diff
@@ -74,6 +74,8 @@ test =
     pytest-cov
     pytest-doctestplus
 pymc3 =
+    jwst==1.11.4
+    stcal>=1.0.0 # Lower limit needed for our create_integration_model function
```

Comment:
This implementation is probably going to cause us at least some headaches, especially since the user isn't going to know that the jwst version will change with different installation options. And also, what happens if you put both `jwst` and `pymc3` in the optional installs, does the `jwst` version that gets installed depend on the order of `jwst` and `pymc3` in the install command (headaches will ensue if so)

Reply:
I didn't try installing with both, good idea.  I assume it'll break since both version requirements cannot be met.

I'd rather deal with a minor headache than with broken code.  After a discussion with STScI's help desk, I learned that some calibration files changed their formatting in 1.11, but the pipeline wasn't made aware of this change for TSO data until 1.12.

I actually think this is the temporary solution that we've bene looking for.  We can freeze the JWST version for `pymc3` users but continue taking advantage of enhancements for other users.  I'm open to relabelling `pymc3` to something else (like 'jwst+pymc3') and updating the documentation to make it more obvious to the users.